setup.py

```diff
@@ -34,6 +34,7 @@
         "django-filer",
         "django-mptt",
         "django-widget-tweaks",
+        "feedparser",
```

Comment:
Good, and now execute `./dev-tools/install.sh` one time to make sure the dependencies end up in the Pipfile.lock

Reply:
Yes, I'm just installing right now, but my pipfile.lock usually  needs some time;)